pkg/ovs/util.go

```diff
@@ -75,3 +76,16 @@ func getIpv6Prefix(networks []string) []string {
 
 	return ipv6Prefix
 }
+
+func matchAddressSetName(asName string) (bool, error) {
+	matched, err := regexp.MatchString(`^[a-zA-Z_.][a-zA-Z_.0-9]*$`, asName)
```

Comment:
We can compile the regular expression in the init function to improve performance.

Reply:
good idea, how about global variables?

```
var addressSetNameRegex = regexp.MustCompile(`^[a-zA-Z_.][a-zA-Z_.0-9]*$`)

func matchAddressSetName(asName string) (bool, error) {
	matched := addressSetNameRegex.MatchString(asName)
	if !matched {
		return false, fmt.Errorf("address set %s must match `[a-zA-Z_.][a-zA-Z_.0-9]*`", asName)
	}

	return true, nil
}
```

Reply:
LGTM, but there is no need to introduce the variable `matched`.